tasks/job.yml

```diff
@@ -19,4 +19,5 @@
     include: "{{ cohesity_protection.include | default('') }}"
     time_zone: "{{ cohesity_protection.time_zone | default('America/Los_Angeles') }}"
     start_time: "{{ cohesity_protection.start_time | default('') }}"
+    delete_sources:  "{{ cohesity_protection.delete_sources | default(False) }}"
```

Comment:
On second thought, can't you create new role/task for this ? create , add sources/update, delete ,

Reply:
Existing modules takes care of addition and updation, new changes is for deletion.